pkg/storage/bucket/swift/config.go

```diff
@@ -26,6 +26,16 @@ type Config struct {
 	MaxRetries        int           `yaml:"max_retries"`
 	ConnectTimeout    time.Duration `yaml:"connect_timeout"`
 	RequestTimeout    time.Duration `yaml:"request_timeout"`
+	HTTPConfig        HTTPConfig    `yaml:"http_config"`
+}
+
+// HTTPConfig stores the http.Transport configuration
+type HTTPConfig struct {
+	Timeout               time.Duration `yaml:"timeout"`
+	IdleConnTimeout       time.Duration `yaml:"idle_conn_timeout"`
+	ResponseHeaderTimeout time.Duration `yaml:"response_header_timeout"`
```

Comment:
These values are define here but I don't see anywhere they are actually being used to configure the underlying transport?